### PR TITLE
docs: connect sdk cosmwasm

### PR DESCRIPTION
**Pull request by technicallyty**
yerrrr

closes: CON-1770

### CODE REVIEW

File: docs/developers/connect-sdk.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+4. price nonce is not 0.
+
+
+### Code Example
+
+```rust contract.rs
+use connect_sdk::bindings::query::ConnectQuery;
+use connect_sdk::bindings::querier::ConnectQuerier;
+use connect_sdk::bindings::marketmap::types::CurrencyPair;
+use cosmwasm_std::{entry_point, Binary, Deps, DepsMut, Empty, Env, MessageInfo, Response, StdResult, to_json_binary, Int128, StdError};
```
Comment (wesl-ee) [inline docs/developers/connect-sdk.mdx]: unused imports: `Binary`, `DepsMut`, `Empty`, `MessageInfo`, `Response`, `entry_point`, and `to_json_binary`

File: docs/developers/connect-sdk.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    let quote = currency_pair.quote.to_uppercase();
+
+    // Check if the market exists and is enabled
+    let market = connect_querier.get_marketmap_market(base.clone(), quote.clone())?;
+    if !market.market.ticker.enabled {
+        return Err(StdError::generic_err("market is not enabled"));
+    }
+
+    // Check price validity
+    let price_response = connect_querier.get_oracle_price(base, quote)?;
```
Comment (wesl-ee) [inline docs/developers/connect-sdk.mdx:73]: error[E0061]: this method takes 1 argument but 2 arguments were supplied
  --> contracts/x-oracle-passthrough/src/contract.rs:22:42
   |
22 |     let price_response = connect_querier.get_oracle_price(base, quote)?;

**Review by wesl-ee — CHANGES_REQUESTED** [review]
L

File: docs/developers/connect-sdk.mdx (hunk 2 above)
Comment (technicallyty) [reply docs/developers/connect-sdk.mdx:73]: ah, im using v0.1.0 for the example, likely an error from v0.2

Comment (technicallyty) [reply docs/developers/connect-sdk.mdx:73]: ill add that im using that version to the doc

File: docs/developers/connect-sdk.mdx (hunk 1 above)
Comment (technicallyty) [reply docs/developers/connect-sdk.mdx]: removed em 🤙🏻

**Review by wesl-ee — APPROVED** [review]

**Pull request merged**